lambda-buffers-compiler/test/Test/KindCheck.hs

```diff
@@ -98,7 +74,7 @@ kcTestOrdering =
     genModuleIn2Layouts = do
       mods <- arbitrary
       shuffledMods <- shuffle mods
-      pure (P.CompilerInput mods, P.CompilerInput shuffledMods)
+      pure (_CompilerInput mods, _CompilerInput shuffledMods)
```

Comment:
Because the `_CompilerInput` now creates a `Map` out of the shuffled list - the two `Maps` might not contain the same `TyDefs` - one map might contain one Module definition and the other another. 

If internally we're now dealing with `Maps` which are guaranteed by the `ProtoCompat` to error out prior if there are two instances of the same `Map` the above test can be removed - as long as it is covered by a test on the ProtoCompat parsing side.

Reply:
This was due to TyName being generated with the same name, so the indexing into Maps makes drastically different Modules. Will introduce this test case in the API tests, it's a good one.